version.go

```diff
@@ -1,4 +1,4 @@
 package chartmogul
 
 // Version of the package
-var Version = "v3"
+var Version = "v4"
```

Comment:
I am guessing the PR should really not be merged into `v3` branch, but rather into a new `v4` branch.